gentest/fixtures/YGAlignBaseline.html

```diff
@@ -0,0 +1,87 @@
+<div id="align_baseline" style="width: 100px; height: 100px; flex-direction:row; align-items: baseline;">
```

Comment:
This whole file would probably make sense to have as part of the YGAlignItems.html + YGAlignSelf.html files which already exist